www/templates/www/users.html

```diff
@@ -5,11 +5,6 @@
 <h2>
   User list
 </h2>
-<ul>
-  <li class="nav-item">
-    <a class="nav-link" href="{% url 'users/create' %}">{% trans 'Create new user' %}</a>
-  </li>
-</ul>
```

Comment:
Placement of such link can always be thought, but we do need an way for admin to add user too, which for this was created. I don't care that much where such feature lives, but it needs to exists.

Reply:
Why? new user can be created by going through the registration form. This simplifies things as the process is always the same.

Reply:
This option has 2 main uses at Vaasa. Main difference in general is needing of approval- Especially helpful when adding plenty of users, usually on deployment stage though, as there is no importer for existing user data from.. well.. much anywhere existing, sans some specific Tampere format.

More broadly speaking while this option os more of band-aid rather than ideal sopution, at least in Vaasa we need an way to add door grants to non-members too (subtenants) and for those we wouldn't even usually care to give real email, not that it directly affects for exacmple approval process...

So.. In the time of now, I'd like to see this feature exist somewhere, until perhaps something better is figured out for either to these points, or the feature itself... First one is important generally at deployment time, latter with streamlining "admin adding user".

Now should this also just get nuked.. I know I can always get around, but root-reasons why this exists as it is currently are those :)

Reply:
ok, lets re-add this in the users list for now until there are better ways to handle this.